books/kestrel/c/syntax/validator.lisp

```diff
@@ -2192,6 +2251,29 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(define make-pointers-to ((pointers typequal/attribspec-list-listp)
```

Comment:
Perhaps we could move this to abstract-syntax-operations.lisp?

Reply:
That would require including `types.lisp` in `abstract-syntax-operations.lisp`, which would create a circular dependency (apparently `types.lisp` indirectly depends on `abstract-syntax-operations.lisp`).

Reply:
Right. Perhaps we might move it to types.lisp then? It just seems a more general operation.